src/main/views/et-fortnightly-list.njk

```diff
@@ -0,0 +1,153 @@
+{% from "govuk/components/table/macro.njk" import govukTable as 'govukTable' %}
+{% from "./macros/common-components.njk" import goBack %}
+
+{% extends "template.njk" %}
+{% block pageTitle %}
+  {{ title }}
+{% endblock %}
+{% block beforeContent %}
+  {{ super() }}
+  {{ goBack(text = backButton) }}
+{% endblock %}
+{% block content %}
+  <div class="parent-box">
+
+    <h2 id="page-heading" class="govuk-heading-l">
+      {{ title1 }}: {{ region }}<br/>
+      {{ title2 }}
+    </h2>
+    <p class="govuk-body govuk-!-font-weight-bold govuk-!-margin-bottom-1">{{ listFor }} {{ contentDate }}</p>
+    <p class="govuk-body">{{ listUpdated | replace("DATE", publishedDate) }} {{ publishedTime }}</p>
+  </div>
+  <details class="govuk-details" data-module="govuk-details" open>
+    <summary class="govuk-details__summary"><span class="govuk-details__summary-text">{{ importantInformation }}</span>
+    </summary>
+    {% set venueEmail = listData['venue']['venueContact']['venueEmail'] %}
+    <div class="govuk-details__text">
+      <p>{{ openJustice1 }}</p>
+      <p>{{ openJustice2 }}</p>
+      <p>{{ openJustice3 }}</p>
+      <p>{{ openJustice4 }}</p>
+      <p>{{ listData['venue']['venueName'] }}, {{ venueEmail }}
```

Comment:
The venue data should be in the same paragraph as openJustice4 followed by the :

Reply:
done